### ISSUE TITLE
Architect should be able to write to a Kettle file based repository

### ISSUE THREAD

**Issue by dibe0014** [SQLPower/power-architect#16]
When generating Kettle jobs, Architect has the option to write the files to a database repository or to the file system. There should also be an option to save the Kettle files to a file based repository.

**Comment by KirtiMistry** [comment]
Hi Benjamin,
Good Morning!

Do you want to add 3rd option to save a kettle job as a file based
repository?
[image: Inline image 1]

Can you please explain me ?

Thanks,

Kirti

**Comment by dibe0014** [comment]
Yes, I would like a third option, to store the kettle job to a file based repository.

**Comment by KirtiMistry** [comment]
Thank you.!

**Comment by KirtiMistry** [comment]
Hi Benjamin!

I was trying create Kettle job using 'Save Job to Repository'. I'm getting
some Pentaho error.
Can you please write me steps to create Kettle job using 'Save Job to
Repository'.

- Kirti

**Comment by KirtiMistry** [comment]
Hi Benjamin,

You mean user can save a job to existing ' File Based Repository' or by creating 'New Folder' inside 'File Based Repository', where 'File Based Repository' already exists/created using Pentaho. Clarify if the requirement is different.

-Kirti
